ldregistry/templates/main.vm

```diff
@@ -2,7 +2,7 @@
 ##
 ## Input parameters:
 ##   $uri     - the requested URI, relative to the registry root (generated by registry-core)
-##   $entity  - the URI of a external entity to be displayed, passed as a query parameter, optional
+##   $entity  - no longer supported
 ##   $_browse - flag (query parameter) to indicate user has selected "browse"  to override landing page, optional
 
 #set( $isLandingPage = $uri.isEmpty() && !$entity && !${_browse} )
```

Comment:
```suggestion
#set( $isLandingPage = $uri.isEmpty() && !${_browse} )
```
The inclusion of this here means entity is not being ignored completely, even if entity is an empty string.